.github/workflows/ci.yml

```diff
@@ -49,6 +50,19 @@ jobs:
         run: |
           $TRAVIS_DIR/install-backend.sh ${{ matrix.BACKEND }}
 
+      - name: Init Mysql Env
```

Comment:
can we init mysql only if necessary?

Reply:
yep, can only init it when backend is `mysql` (better init it in `install-mysql-docker.sh`, but it doesn't work well)